wordpress-mobile/WordPress-iOS
# Quick Start V2: update checklist

Refs. #10860 

This PR updates the current quick start checklist. It uses 2 sections to divide the completed tasks from the todo tasks.

![qs-section](https://user-images.githubusercontent.com/912252/51679828-e8762f80-1fd7-11e9-8e54-eb62b0531347.gif)

Because this update is behind feature flag, I created a new data flow _QuickStartChecklistManager_ to handle the table view datasource and delegate.

@SylvesterWilmott this is the final result:

![simulator screen shot - iphone x - 2019-01-24 at 13 52 16](https://user-images.githubusercontent.com/912252/51682542-91745880-1fdf-11e9-9018-e8ae9225fd52.png)

![simulator screen shot - iphone x - 2019-01-24 at 13 52 18](https://user-images.githubusercontent.com/912252/51682552-96d1a300-1fdf-11e9-97f8-8eba25cb986b.png)


**To test:**
- Create a new site and turn on quick start (or put return true in shouldShowQuickStartChecklist())
- Select _Customize Your Site_ to complete at least one task
- You should see the todo list separated from the completed list
- Tap on the _Complete_ header to expand/hide the section
